### ISSUE TITLE
Per Game bios Selection

### ISSUE THREAD

**Issue by RevQuixo** [RevQuixo/MegaSD-Bugtracker#8]
Add a bios file to the game folder to override the default bios. Useful for games that don't like "standard" bios choices. Possibly rename the bios so it doesn't trigger MD+

**Comment by TheShadowRunner** [comment]
Definitely, this is already implemented by the competition and it works beautifully (Altered Beast PC Engine CD..)

Rev do you think there's a chance neodev comes around these parts? Is he aware of this tracker ?
